scripts/deploy.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -ev
+testchange=`git diff --exit-code -- docs > /dev/null`
```

Comment:
Is the rationale that if there's an actual diff in the `docs` directory, then something meaningful changed, and therefore it's worth going through with the deploy?

If so, this might be worth a comment, because Future Jeff / Future Ralph / Future Someone might wonder why this two-step process is here.

Reply:
Per @ribab:

> Only run gh-pages travisci IF something in mtt/docs changes

Reply:
I can add a comment - my question to @ggouaillardet was: does this work the way I think it should?